src/constants/common.ts

```diff
@@ -22,21 +22,25 @@ export const LOCAL_STORAGE_PREFIX = 'sh-wallet';
 
 export const PROTOCOL_AETERNITY = 'aeternity';
 export const PROTOCOL_BITCOIN = 'bitcoin';
+export const PROTOCOL_DOGE = 'dogecoin';
 
 export const PROTOCOLS = [
   PROTOCOL_AETERNITY,
   PROTOCOL_BITCOIN,
+  PROTOCOL_DOGE,
```

Comment:
We won't support Doge for now. Please remove this lilne as it will force us to write things that don't matter for now.